src/Menubar.tsx

```diff
@@ -172,23 +193,73 @@ function MenuAdvanced({ openPanel }: IPanelProps) {
           onChange={(x) => setFramerate(x)}
         />
       </Label>
+       <MenuItem
+        icon="media"
+        text="Video settings"
+        onClick={() => openPanel({ component: MenuPicture, title: "Video settings" })}
+      />
+      <MenuDivider />
       <Button icon="wrench" text="Factory reset" fill />
       <Button icon="updated" text="Update" fill />
```

Comment:
It mixed MenuItems (without Menu! I don't think we can do that even though it doesn't complain :thinking: ) and Buttons. We should fix that